Contributor Documentation/LSP Extensions.md

```diff
@@ -595,30 +595,3 @@ export interface PeekDocumentsResult {
   success: boolean;
 }
 ```
-
-## `workspace/getReferenceDocument`
```

Comment:
The `LanguageServerProtocol` module currently represents LSP 3.17, so I would still consider  `workspace/textDocumentContent` an extension. Just one that happens to line up with 3.18. Could you add that to the `LSP Extensions` document and mention that it matches the 3.18 request?